gc_little_helper_II.user.js

```diff
@@ -2286,40 +2288,41 @@ var mainGC = function() {
     }
 
 // Improve event date and event time in cache listing.
-    if (is_page("cache_listing") && $('#cacheDetails svg.cache-icon use')[0] && $('#cacheDetails svg.cache-icon use')[0].href.baseVal.match(/\/cache-types.svg\#icon-(6$|6-|453$|453-|13$|13-|7005$|7005-|3653$|3653-)/)) {  // Event, MegaEvent, Cito, GigaEvent, CommunityCelebrationEvents
-        // Show eventday beside date.
-        if (settings_show_eventday) {
-            try {
-                var match = $('meta[name="og:description"]')[0].content.match(/([0-9]{2})\/([0-9]{2})\/([0-9]{4})/);
-                if (match == null) {
-                    match = $('meta[name="description"]')[1].content.match(/([0-9]{2})\/([0-9]{2})\/([0-9]{4})/);
-                }
-                if (match != null) {
-                    var date = new Date(match[3], match[1]-1, match[2]);
-                    if (date != "Invalid Date") {
-                        var name = " (" + date.getWeekday() + ") ";
-                        var elem = document.createTextNode(name);
-                        var side = $('#ctl00_ContentBody_mcd2')[0];
-                        side.insertBefore(elem, side.childNodes[1]);
-                    }
-                }
-            } catch(e) {gclh_error("Show eventday beside date",e);}
-        }
-        // Show eventtime in 24 hours format.
-        if (settings_show_eventtime_with_24_hours) {
-            try {
-                if ($('#ctl00_ContentBody_mcd2') && $('#mcd3')[0] && $('#mcd4')[0]) {
-                    let sStr = $('#mcd3')[0].innerHTML.trim().match(/^(\D+):\s+(\d{1,2}:\d{2}\s+(AM|PM))$/i);
-                    let eStr = $('#mcd4')[0].innerHTML.trim().match(/^(\D+):\s+(\d{1,2}:\d{2}\s+(AM|PM))$/i);
-                    if (sStr && sStr.length == 4 && eStr && eStr.length == 4) {
-                        var t = convert12To24Hour(sStr[2]);
-                        $('#mcd3')[0].innerHTML = $('#mcd3')[0].innerHTML.replace(sStr[2], t);
-                        var t = convert12To24Hour(eStr[2]);
-                        $('#mcd4')[0].innerHTML = $('#mcd4')[0].innerHTML.replace(eStr[2], t);
-                    }
-                }
-            } catch(e) {gclh_error("Show eventtime in 24 hours format",e);}
-        }
+    if (is_page("cache_listing") && $('h2.CacheDescriptionHeader')[0] && $('#cacheDetails svg.cache-icon use')[0] && $('#cacheDetails svg.cache-icon use')[0].href.baseVal.match(/\/cache-types.svg\#icon-(6$|6-|453$|453-|13$|13-|7005$|7005-|3653$|3653-)/) &&  // Event, MegaEvent, Cito, GigaEvent, CommunityCelebrationEvents
```

Comment:
The Block Party (4738) is missing in the list of event types (example [GCAJ960](https://coord.info/GCAJ960))